_includes/blog_modal.html

```diff
@@ -0,0 +1,24 @@
+<div class="container">
+    <div class="modal fade" id="blog-modal" role="dialog">
+        <div class="blog-modal modal-dialog modal-lg">
+            <div class="modal-content">
+                <div class="modal-body">
+                    <img data-toggle="modal" data-target="#blog-modal" src="{{ site.baseurl }}/assets/images/{{include.blog-image}}" width="100%">
+                </div>
+                <div class="modal-footer">
+                    <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
```

Comment:
nit - can you indent by 2 spaces instead of 4 so it's consistent with the rest of the site?